tests/UI/pages/FO/FObasePage.js

```diff
@@ -313,18 +316,43 @@ class FOBasePage extends CommonPage {
     await this.clickAndWaitForNavigation(page, this.cartLink);
   }
 
+  /**
+   * Check if there are autocomplete search result
+   * @param page {Page} Browser tab
+   * @param productName {string} Product name to search
+   * @returns {Promise<boolean>}
+   */
+  async hasAutocompleteSearchResult(page, productName) {
+    await this.setValue(page, this.searchInput, productName);
+    await page.waitForTimeout(2000);
```

Comment:
If there is a way, please remove this line. we like to avoid it as much as possible

Reply:
Fixed